### DISCUSSION TITLE
Add podman Support in Lieu of Docker

### DISCUSSION THREAD

**Discussion by ColtAllen** [PrefectHQ/prefect#15836] [Ideas]
Docker now requires a commercial license. Are there any plans to support podman, the OSS fork of Docker?

![image](https://github.com/user-attachments/assets/dcdee155-e06d-43e6-9d9e-54a688c5620c)

https://podman.io/

**Comment by Daniel-Ross** [comment]
I'm curious about this as well.

**Comment by zzstoatzz** [comment]
hi @ColtAllen and @Daniel-Ross - I think it makes sense to explore this. I'm not what disparities exist, I assume podman covers all the core functionality we need from docker in the worker etc - any further details from podman aficionados appreciated

**Reply by wcc-duckie** [reply]
This is interesting to me as well. Our organization very much would like us to move from docker to podman.
